source/index-management.rst

```diff
@@ -185,6 +204,44 @@ Standard API
     listIndexes(): Cursor;
   }
 
+  interface CreateIndexOptions {
+    /**
+     * Specifies how many data-bearing members of a replica set, including the primary, must
+     * complete the index builds successfully before the primary marks the indexes as ready.
+     *
+     * This option accepts the same values for the "w" field in a write concern plus "all",
+     * which indicates all voting data-bearing nodes.
+     *
+     * @note This option is only available in MongoDB 4.4+.
+     *
+     * Drivers MUST NOT throw an error if this option is specified when creating an index on
+     * an earlier version of the server. Instead, drivers MUST rely on the server for returning
+     * an appropriate error.
+     *
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     */
+    commitQuorum: Optional<Int32 | String>;
+
+    /**
+     * The maximum amount of time to allow the index build to take before returning an error.
+     *
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     */
+    maxTimeMS: Optional<Int64>;
```

Comment:
I included this option here since we had a [DRIVERS ticket](https://jira.mongodb.org/browse/DRIVERS-374) ensuring all drivers already support it on the creation / dropping helpers. The ticket didn't ensure the listing helpers had it though, so I didn't include an options definition for those helpers. 

The Rust and Ruby drivers appear to be the only ones that don't actively support this as of yet (we just gotten to it yet in Rust, and Ruby didn't [properly test it](https://jira.mongodb.org/browse/RUBY-1218?focusedCommentId=1595722&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1595722) with the driver's API when they closed this ticket).